src/models/config.ts

```diff
@@ -6,30 +6,25 @@
 import { grpc } from '@improbable-eng/grpc-web'
 
 /**
- * @hidden
+ * Configuration for Threads DB Client.
  */
-export interface BaseConfig {
+export interface Config {
```

Comment:
This just means Config can be pretty much anything, but it _must_ have a toJSON method.